include/solvers/navier_stokes_base.h

```diff
@@ -118,6 +118,11 @@ class NavierStokesBase : public PhysicsSolver<VectorType>
   {
     return nonzero_constraints;
   };
+  double
+  get_current_residual() override
+  {
+    return system_rhs.l2_norm();
+  }
```

Comment:
Is this really a change or it was already like that? Is this function necessary?